marklogic-data-hub-central/src/main/java/com/marklogic/hub/central/controllers/steps/MergingStepController.java

```diff
@@ -96,9 +96,22 @@ public ResponseEntity<JsonNode> validateMergingStep(@PathVariable String stepNam
         return ResponseEntity.ok(MasteringService.on(getHubClient().getFinalClient()).validateMergingStep(stepName, view, entityPropertyPath));
     }
 
+    @RequestMapping(value = "/merge", method = RequestMethod.PUT)
+    @Secured("ROLE_readMerging")
+    public ResponseEntity<JsonNode> mergeDocument(@RequestParam List<String> mergeURIs, @RequestParam String flowName, @RequestParam(required = false)  String stepNumber, @RequestParam(required = false, defaultValue = "false") Boolean preview, @RequestParam(required = false, defaultValue = "{}") String options) {
```

Comment:
@briantang There is already another mergeDocument method with the same signature and implementation. You can remove this.

Reply:
I removed the method and force pushed the changes